### ISSUE TITLE
Add tabs below titlebar

### ISSUE THREAD

**Issue by bhajneet** [ShabadOS/desktop#155]
Related to #129 

There should be tabs for different navigators. In our current set up each presenter has it's own controller embedded inside it. So each tab should be changing the controller and therefore the presenter as well.

Mockup: 

![image](https://user-images.githubusercontent.com/14130567/55505497-3aaf7e80-5621-11e9-857f-64203aa29ecf.png)

These tabs should hide away like the titlebar does. If there is no mouse activity, just hide. This way the projector will continue to look as clean as possible most of the time, but allows users to quickly change between shabads.

Related to #154 by allowing right click on search/bookmark/history items to open in a new tab. Can also have a setting that allows all activated searches to open as a new tab (Effectively becoming a tab row of history). This means one could be presenting asa di var, they search a shabad, new tab opens up, once done they can either ctrl+w to close tab or ctrl+tab to resume asa di var.

**Comment by Harjot1Singh** [comment]
Is the controller separate for each tab? How does this play out with the mobile controller?

**Comment by bhajneet** [comment]
Yes, separate controller. I don't know how else it would work.

Mobile would have a collapsed tab button which shows multiple tabs as list items inside it.

Need to have some special/dynamic function tabs as well like "Active Line" which has it's display/controller change based on what is activated

**Comment by Harjot1Singh** [comment]
Yep, I think we should flesh some of these out and links as related issues.

**Comment by bhajneet** [comment]
Can revisit after both alphas are complete

**Comment by bhajneet** [comment]
On hold until mobile tabs are implemented and tested
